service/controller/v1/key/key.go

```diff
@@ -63,6 +63,10 @@ func APIServiceHost(prefix string, clusterID string) string {
 	return fmt.Sprintf("%s.%s:443", prefix, clusterID)
 }
 
+func ManagedAppPodMetricsPath() string {
```

Comment:
const or function like this? I prefer to function as it is.

Reply:
Key function makes sense to me. But there are no values so why are you using `Sprintf`?

Reply:
Old codes are left, I will remove it, thanks.